apps/sms/views/conversation/js/conversation.js

```diff
@@ -745,37 +740,36 @@ var ConversationView = {
     // We'll revisit this.draft necessity in bug 1164435.
     this.draft = Drafts.byDraftId(draftId);
 
-    // Draft recipients are added as the composer launches
-    if (this.draft) {
-      // Recipients will exist for draft messages in threads
-      // Otherwise find them from draft recipient numbers
-      this.draft.recipients.forEach(function(number) {
-        Contacts.findByAddress(number).then(function(contacts) {
-          var recipient;
-          if (contacts.length) {
-            recipient = Utils.basicContact(number, contacts[0]);
-            recipient.source = 'contacts';
-          } else {
-            recipient = {
-              number: number,
-              source: 'manual'
-            };
-          }
+    if (!this.draft) {
+      return;
+    }
 
-          this.recipients.add(recipient);
-        }.bind(this));
-      }, this);
+    // Recipients will exist for draft messages in threads
+    // Otherwise find them from draft recipient numbers
+    this.draft.recipients.forEach(function(number) {
+      Contacts.findByAddress(number).then(function(contacts) {
+        var recipient;
+        if (contacts.length) {
+          recipient = Utils.basicContact(number, contacts[0]);
+          recipient.source = 'contacts';
+        } else {
+          recipient = {
+            number: number,
+            source: 'manual'
+          };
+        }
+
+        this.recipients.add(recipient);
+      }.bind(this));
+    }, this);
 
-      // Render draft contents into the composer input area.
-      Compose.fromDraft(this.draft);
+    // Render draft contents into the composer input area.
+    Compose.fromDraft(this.draft);
 
-      // Discard this draft object and update the backing store
-      Drafts.delete(this.draft).store();
+    // Discard this draft object and update the backing store
+    Drafts.delete(this.draft).store();
 
-      this.draft.isEdited = false;
-    } else {
-      this.recipients.focus();
-    }
+    this.draft.isEdited = false;
```

Comment:
mostly removed the "if" to remove the indentation + removed the "focus" call.
(that's why I didn't rewrite the use of `bind`)